<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Регистрация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>Логин:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="login_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="password_label">
       <property name="text">
        <string>Пароль:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_lineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="password_confirm_label">
       <property name="text">
        <string>Подтвердите парлоь:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="password_confirm_lineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>Имя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="surname_label">
       <property name="text">
        <string>Фамилия:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surname_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="patronymic_label">
       <property name="text">
        <string>Отчество:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="patronymic_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="birthday_label">
       <property name="text">
        <string>Дата рождения:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="birthday_dateEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="phone_number_label">
       <property name="text">
        <string>Номер телефона:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phone_number_lineEdit">
       <property name="inputMask">
        <string>+\7(\999)-999-99-99</string>
       </property>
       <property name="text">
        <string>+7(9)---</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>Электронная почта:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="error_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="sign_up_pushButton">
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
